test/tests/functional/pbs_qstat_dsv.py

```diff
@@ -55,15 +55,28 @@ def parse_dsv(self, attrs_qstatf, jid, qstat_type, delimiter=None):
             delim = " "
         if qstat_type == "job":
             cmd = ' -f -F dsv ' + delim + " " + str(jid)
+            qstat_cmd_dsv = os.path.join(self.server.pbs_conf['PBS_EXEC'],
+                                         'bin', 'qstat') + cmd
             qstat_cmd = os.path.join(self.server.pbs_conf['PBS_EXEC'],
-                                     'bin', 'qstat') + cmd
+                                     'bin', 'qstat') + ' -f ' + str(jid)
```

Comment:
Good point.  I had forgotten about the change of keys.  Keeping as direct calls to qstat seems best.